graphql/ui_plugin_build_baron.go

```diff
@@ -115,21 +115,32 @@ func BbGetCreatedTicketsPointers(taskId string) ([]*thirdparty.JiraTicket, error
 	return results, nil
 }
 
-func GetSearchReturnInfo(taskId string, exec string) (*thirdparty.SearchReturnInfo, bool, error) {
-	projectNotFoundError := false
+type buildBaronConfig struct {
+	ProjectFound     bool
+	SearchConfigured bool
+}
+
+func GetSearchReturnInfo(taskId string, exec string) (*thirdparty.SearchReturnInfo, buildBaronConfig, error) {
+	bbConfig := buildBaronConfig{}
 	t, err := BbGetTask(taskId, exec)
 	if err != nil {
-		return nil, projectNotFoundError, err
+		return nil, bbConfig, err
 	}
 	settings := evergreen.GetEnvironment().Settings()
 	buildBaronProjects := BbGetConfig(settings)
 	bbProj, ok := buildBaronProjects[t.Project]
 
 	if !ok {
-		projectNotFoundError = true
-		return nil, projectNotFoundError, errors.New(fmt.Sprintf("Build Baron project for %s not found", t.Project))
+		return nil, bbConfig, errors.Errorf("Build Baron project for %s not found", t.Project)
 	}
+	bbConfig.ProjectFound = true
 
+	// the build baron is configured if the jira search is configured
+	if len(bbProj.TicketSearchProjects) <= 0 {
+		bbConfig.SearchConfigured = false
```

Comment:
this seems like another instance of what Annie was pointing out.

Reply:
Yeah, but I decided to keep if for increased readability. I'm open to removing it if you think that's better though.

Reply:
🤷‍♂️